ref-exchange/src/rated_swap/stnear.rs

```diff
@@ -0,0 +1,85 @@
+use near_sdk::{ext_contract, json_types::U128, near_bindgen, PromiseOrValue};
```

Comment:
For a pool type that not serves a specific swap pair stnear-near only, we'd better wrap up their difference in some way. The difference is mainly in the price/rate related logic, more specific, is how to update a token's rates, and to tell a token's rate currently is valid or not. 

If we can abstract those two methods, provide uniform access interface to RatedSwapPool structure, and enclose their implements in different mod files like this one, it would be easy to add more rated stable pool in future.